src/main/java/frc/robot/RobotContainer.java

```diff
@@ -140,6 +140,7 @@ private void configureButtonBindings() {
     if (Robot.armEnabled) {
       OI.Driver.getArmHighButton().onTrue(new ProxyCommand(() -> armSubsystem.handleHighButton()));
       OI.Driver.getArmMidButton().onTrue(new ProxyCommand(() -> armSubsystem.handleMidButton()));
+      OI.Operator.getArmStopButton().onTrue(new InstantCommand(() -> armSubsystem.cancelCommands()));
```

Comment:
Please add .withName so we can tell the current command that is running instead of "InstantCommand"

Reply:
done, name has been added